examples/pico/book/src/main/java/io/helidon/examples/pico/book/BlueColor.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.examples.pico.book;
+
+import io.helidon.pico.Contract;
+
+@Contract
+@org.jvnet.hk2.annotations.Service
+@jakarta.inject.Singleton
+@jakarta.inject.Named
```

Comment:
Illustrating how Pico can be used with javax or jakarta - i don't want to assume our users understand this.  Better to see it in black and white.

Reply:
hmm that would suggest, that our users do not understand imports. I might be worried, they might take it as a fact, that it has to look like this rather then that it is there for them to better understand it. But if you believe, this will be really more beneficial, lets leave it as it is.